data/import/common/common/cms_block.csv

```diff
@@ -35,8 +35,8 @@ cms-block-email--availability-notification-unsubscribed--html,availability-notif
so-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;"">                     <table class=""sprykerTextContentContainer"" style=""min-width: 100% !important;background-color: #F9F9F9;border-collapse: collapse;mso-table-lspace: 0pt;mso-table-rspace: 0pt;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;"" width=""100%"" cellspacing=""0"" border=""0"">                       <tbody>                         <tr>                           <td class=""sprykerTextContent"" style=""padding-top: 18px;padding-right: 0px;padding-bottom: 0px;padding-left: 9px;font-family:Helvetica, Arial, Verdana, sans-serif;font-size: 15px;font-weight: normal;text-align: left;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;word-break: break-word;"" valign=""top"">                             <p style=""color: #1EBEA0;""><strong>{{ item.name }} - {{ 'customer.order.item_sku' | trans }}: {{ item.sku }}</strong></p>                             <p style=""color: #4A4A4A"">{{ 'mail.trans.order.quantity' | trans }}: {{ item.quantity | executeFilterIfExists('formatInt', app.locale) }}</p>                           </td>                           <td class=""sprykerTextContent"" style=""padding-top: 18px;padding-right: 9px;padding-bottom: 9px;padding-left: 0px;min-width: 100px;color: #000000;font-family:Helvetica, Arial, Verdana, sans-serif;font-size: 15px;font-weight: normal;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;word-break: break-word;"" valign=""top"">                             <p style=""text-align: right;""><strong>{{ item.sumPrice | money(true, mail.order.currencyIsoCode) }}</strong></p>                           </td>                         </tr>                         <tr>                           <td class=""sprykerTextContent"" colspan=""2"" style=""padding-top: 0px;padding-right: 0px;padding-bottom: 9px;padding-left: 9px;font-family:Helvetica, Arial, Verdana, sans-serif;font-size: 15px;font-weight: normal;text-align: left;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;word-break: break-word;"" valign=""top"">                             <!--{% for option in item.productOptions %}-->                               <p style=""color:#8F8F8F"">+ {{ option.groupName }}: {{ option.sumPrice | money(true, mail.order.currencyIsoCode) }}</p>                             <!--{% endfor %}-->                           </td>                         </tr>                       </tbody>                     </table>                   </td>                 </tr>               <!--{% endfor %}-->             <!--{% endfor %}-->           </tbody>         </table>         <!--[if mso]>         </td>         <![endif]-->         <!--[if mso]>         </tr>         </table>         <![endif]-->       </td>     </tr>   </tbody> </table> <!--[if (gte mso 9)|(IE)]> </td> </tr> </table> <![endif]--> <!-- // END TEMPLATE -->"
```

Comment:
Only DynamicStore-related variables must be changed, but, here a lot more stuff changed. (At least image URLs)

Please fix.